### PR TITLE
fix: update router content in cybersecurity roadmap

### LINKED ISSUE #7460: Cyber-Security/Router wrong content
### Detailed Description

The content for `Router` seems to be a copy&paste from S3. 

## Content in Question:

>Amazon Simple Storage Service (S3) is a scalable, object-based cloud storage service provided by AWS. It allows users to store and retrieve large amounts of data, such as files, backups, or media content, with high durability and availability. S3 is designed for flexibility, enabling users to access data from anywhere via the internet while offering security features like encryption and access controls. It is widely used for data storage, content distribution, disaster recovery, and big data analytics, providing cost-effective, scalable storage for a variety of applications.

### PR DESCRIPTION
**Pull request by kabeer05**
Updated content to Cyber-Security Roadmap:
Fix #7460

- Router

(PR for Hacktoberfest)

### CODE REVIEW

File: src/data/roadmaps/cyber-security/content/router@lwSFIbIX-xOZ0QK2sGFb1.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
+A router is a networking device that directs data packets between different networks, ensuring they reach their destination. It operates at the network layer (Layer 3) of the OSI model and forwards data based on the IP addresses of the source and destination. Routers are essential for connecting devices to the internet or linking multiple networks together. They maintain a routing table to decide the best path for data and can dynamically update routes using protocols like RIP, OSPF, or BGP. Routers also handle Network Address Translation (NAT), allowing multiple devices to share a single public IP address. Many modern routers offer Wi-Fi for wireless connectivity and include basic firewall security to protect the network from threats.
 
 Learn more from the following resources:
 
 - [@article@What is a Router](https://www.cloudflare.com/en-gb/learning/network-layer/what-is-a-router/)
 - [@video@What is a router and how does it work?](https://www.youtube.com/watch?v=UIJzHLpG9bM)
-
+- [@video@Everything Routers do - Part 1 - Networking Fundamentals - Lesson 5](https://youtu.be/AzXys5kxpAM?si=nEsCH6jG2Lj6Ua8N)
```
Comment (offensive-vk) [inline src/data/roadmaps/cyber-security/content/router@lwSFIbIX-xOZ0QK2sGFb1.md:9]: No need to mention the entire video title. Just keep it short and concise.

**Review by offensive-vk — COMMENTED** [review]
Left some refractors.

File: src/data/roadmaps/cyber-security/content/router@lwSFIbIX-xOZ0QK2sGFb1.md (hunk 1 above)
Comment (kabeer05) [reply src/data/roadmaps/cyber-security/content/router@lwSFIbIX-xOZ0QK2sGFb1.md]: Fixed it

**Comment by dansholds** [comment]
LGTM but this doesn't satisfy hacktoberfest acceptance, please check the contribution docs.

**Pull request merged**